### DISCUSSION TITLE
Fehler bei Auswahl von Wallbox Typen

### DISCUSSION THREAD

**Discussion by SunnyHomeManager** [camueller/SmartApplianceEnabler#195] [Installation und Betrieb]
Wenn ich die Wallboxen Keba, Phoenix oder Wallbe auswähle schließt sich nicht das Auswahlfenster und es werden auch unter dem Fenster für Lesezugriff und Schreibzugriff nur zwei kleine Kästchen angezeigt (siehe Bild). 

Der go-eCharger und WARP Charger kann dagegen geladen werden.

![image](https://user-images.githubusercontent.com/99254790/153201093-3ea9805f-c00e-4882-b7b9-ea1e9877c018.png)

**Comment by camueller** [comment]
Das sind genau die Wallboxen, die über Modbus angesprochen werden. Offensichtlich hast Du noch keinen Modbus/TCP im SAE konfiguriert, was bei meinen automatisierten oder auch manuellen Tests immer der Fall ist. Aber "hängen" soll die Web-Oberfläche deswegen nicht - insofern werde ich das für die nächste Version fixen. Muss nur noch über legen, welcher Fix sinnvoll ist.

**Comment by camueller** [comment]
Ist gefixt für die nächste Version. Das Feld "Modbus" ist bei Auswahl einer Vorlage für Modbus-Wallboxen bereits Pflichfeld, d.h. man muss einen Wert selektieren. Damit man weiss, was zu tun ist, wenn die Liste leer ist, wird jetzt ein Tooltip angezeigt.
